config/environments/development.rb

```diff
@@ -73,4 +73,5 @@
   # Allow incoming connections over ngrok
   config.hosts << /[a-z0-9]+\.ngrok\.io/
   config.offseason = false
+  config.ngrok_url = "http://cec1d9b787e4.ngrok.io"
```

Comment:
Delete ngrok_url for now

Reply:
Consider adding `config/environments/development_overrides.rb` that's in `.gitignore`.

If we go that route, we can add a comment to _this_ file suggesting you add a `config.ngrok_url` to _that_ (development_overrides).